libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -171,7 +171,7 @@ data TeamMember = TeamMember
 
 newtype TeamMemberList = TeamMemberList
     { _teamMembers :: [TeamMember]
-    }
+    } deriving (Monoid)
```

Comment:
Do we actually use/need this `instance` ?

Reply:
yes. It's needed for the _Just getter, making it simpler to convert the empty memberlist to []